packages/cli/src/commands/generate/cell/__tests__/__snapshots__/cell.test.js.snap

```diff
@@ -169,7 +169,7 @@ export const Success = ({ userProfile }) => {
 
 exports[`creates a cell mock with a camelCase word name 1`] = `
 "// Define your own mock data here:
-export const standard = (/* vars, { ctx, req } */) => ({
```

Comment:
I dunno why the snapshots do this weird thing. Its not actually doing this when we actually generate 🤷‍♂️